package.json

```diff
@@ -18,7 +18,7 @@
     "discord.js": "^13.1.0",
     "express": "^4.17.1",
     "lodash": "^4.17.21",
-    "mongoose": "^6.0.8",
```

Comment:
Not necessary update mongoose version to fix that issue

Reply:
This is from me forking main, should I have forked dev instead?

Reply:
Oh, i didnt notice

Fixed on #10